packages/base-styles/_colors.scss

```diff
@@ -31,7 +31,7 @@ $medium-gray-text: #757575;
 $light-gray-secondary: #ccc;
 $light-gray-tertiary: #e7e8e9;
 $theme-color: theme(button);
-$blue-medium-focus: #007cba;	// @todo: Currently being used as "spot" color. Needs to be considered in context of themes.
+$blue-medium-focus: #3858e9;	// @todo: Currently being used as "spot" color. Needs to be considered in context of themes.
```

Comment:
Let's try to find a way to compute this based on "theme" (otherwise the alternative schemes are a bit broken)

Reply:
I pushed a fix. There really aren't that many shades of blue used here, but what shades there are should take from this now, right?